### PR TITLE
Add a new target KakuteF4wing

### PR DESCRIPTION
**Pull request by jamming**
KakuteF4wing is made smaller and lighter. It cover an area of  25x30mm. 
1- 6 PWM channel are: 2 motor pads and 4 servo with THT
2- 5 serial ports, one with  inverter on the RX of a UART for traditional SBUS
3- I2C
4- analog OSD
5- Independent 5V 3A BEC for servo
6-16MB onboard flash memory
7- Based on STM32F405 
8- Simple PDB with 2BECs (5V/3A)
9-2 ADC
10- Shunt resistance up to 90A.

### CODE REVIEW

**Comment by MrD-RC** [comment]
Please note that new hardware needs to follow the New Hardware policy. 

https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

**Comment by mmosca** [comment]
@jamming Are you representing the manufacturer and plan to make it an official target, or are you a user trying to submit an unofficial target?

As MrD mentioned, please have a look at the New Hardware policy. For official targets, samples need to be submitted to the team for validation. Let me and MrD know if you need help contacting the dev team. We can both be reached on INAV's discord as well.

**Comment by vincentpoont2** [comment]
Hi @MrD-RC @mmosca, @jamming and I are with Holybro, manufacturer of the Kakute Flight Controllers. We want to get this target into INAV. Is there a specific group in INAV discord for these topic? Or what is your discord ID? My Discord ID is `vincepoon`

**Comment by MrD-RC** [comment]
> Hi @MrD-RC @mmosca, @jamming and I are with Holybro, manufacturer of the Kakute Flight Controllers. We want to get this target into INAV. Is there a specific group in INAV discord for these topic? Or what is your discord ID? My Discord ID is `vincepoon`

Hi Vince,

The best place would be the INAV Discord server and the inav-development channel. https://discord.gg/r5XFzNUE

Both @mmosca and myself are there 👍🏻

**Comment by vincentpoont2** [comment]
> > Hi @MrD-RC @mmosca, @jamming and I are with Holybro, manufacturer of the Kakute Flight Controllers. We want to get this target into INAV. Is there a specific group in INAV discord for these topic? Or what is your discord ID? My Discord ID is `vincepoon`
> 
> Hi Vince,
> 
> Awesome. It will be great to see more Holybro FW flight controllers in INAV.
> 
> The best place would be the INAV Discord server and the inav-development channel. https://discord.gg/r5XFzNUE
> 
> Both @mmosca and myself are there 👍🏻

Thanks! Already made a post few hours ago.

File: src/main/target/KAKUTEF4WING/target.h [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+// ********** External Devices On I2C2******
+#define TEMPERATURE_I2C_BUS     BUS_I2C2
+#define PITOT_I2C_BUS           BUS_I2C2
+#define USE_RANGEFINDER
+#define RANGEFINDER_I2C_BUS     BUS_I2C2
+
+// ********** Internal BARO On I2C2*********
+#define USE_BARO
+#define BARO_I2C_BUS            BUS_I2C2
+#define USE_BARO_BMP280
```
Comment (mmosca) [inline src/main/target/KAKUTEF4WING/target.h:62]: You can also use USE_BARO_ALL here, if you would like.

**Pull request merged**